generator/bindata.go

```diff
@@ -1,44 +1,42 @@
-// Code generated by go-bindata. DO NOT EDIT.
+// Package generator Code generated by go-bindata. (@generated) DO NOT EDIT.
```

Comment:
Could you update your bindata generator? we use Burke's fork

Reply:
How do I achieve that? I ran `brew install go-bindata` but running `go generate ./generator` doesn't make a difference.

Reply:
`go get -u github.com/kevinburke/go-bindata/go-bindata`